definition-map.js

```diff
@@ -14,6 +14,10 @@ const QUERIES = {
             namespace: "test test test"
         }
     },
+    LanguagesQuery: {
+        metadescription: "returns an array of information about available languages",
```

Comment:
Returns*

"Array of information" feels a little vague for documentation purposes. Maybe "array of Language objects configured in the Thought Industries instance"?